Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -298,12 +296,24 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
 - (void)toxAV:(OCTToxAV *)toxAV audioBitRateChanged:(OCTToxAVAudioBitRate)bitrate stable:(BOOL)stable friendNumber:(OCTToxFriendNumber)friendNumber
 {
-    // Lower bitrate if unstable?
+    NSArray *validBitrates = @[@8, @16, @24, @32, @48];
```

Comment:
They were working when I was testing them, well at least for 24/32/48. I never got down lower than 24. These are just numbers that mannol suggested me to try.